README.md

```diff
@@ -17,6 +17,20 @@ Build things 🎉
 
 Before contributing, please make sure that you read our [Contributing Guidelines](https://github.com/twilio-labs/paste/blob/master/CONTRIBUTING.md) and agree with our [Code of Conduct.](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md)
 
+## Linking Local Paste to an Application
+  Sometimes you need to make changes and test them locally. Figuring out all the requirements to do this can often be a nightmare.  So we've set-up a small guide and some scripts to make it easier for you.
```

Comment:
```suggestion
  Sometimes you need to make changes and test them locally. Figuring out all the requirements to do this can often be challenging.  So we've set-up a small guide and some scripts to make it easier for you.
```

Reply:
updated